sdks/python/apache_beam/runners/dataflow/dataflow_runner.py

```diff
@@ -87,7 +87,7 @@ def _get_unique_step_name(self):
     return 's%s' % self._unique_step_id
 
   @staticmethod
-  def poll_for_job_completion(runner, result):
+  def poll_for_job_completion(runner, result, duration):
     """Polls for the specified job to finish running (successfully or not)."""
```

Comment:
<!--new_thread; commit:5b7323ad9e10a5693ea41a239c8a675139b977de; resolved:0-->
Can you please add argument docstrings?  It is unclear from a first read what should be passed to `duration`, especially given that on line 112, there is an `if condition` branch.  Do we want to make this an optional kwarg?

Reply:
Added the doc strinng. I would prefer not to use optionals args, since there is only one consumer of this method.